sdcm/cluster.py

```diff
@@ -3242,6 +3242,14 @@ def nodes_by_region(self, nodes=None) -> dict:
             grouped_by_region[node.region].append(node)
         return grouped_by_region
 
+    def datacenter_name_per_region(self):
```

Comment:
we don't change datacenters during the test. `@cache`? 
Also if it's method, add `get_` prefix to it, or use it as` @property` or `@cached_property`

Reply:
We have nemesis that add new datacenter, for example

Reply:
renamed the function